### ISSUE TITLE
copydoc issue with namespace assignment

### ISSUE THREAD

**Issue by EilifTS** [doxygen/doxygen#11045]
**Describe the bug**
Doxygen will fail to resolve a copydoc reference when the referenced class has been assigned as a namespace in another namespace.

**To Reproduce**
Minimal reproducer: [reproducer.txt](https://github.com/user-attachments/files/16469356/reproducer.txt)
The reproducer will give:
> (Path to file):34: error: @copybrief or @copydoc target 'ClassA::TestA()' not found

**Expected behavior**
The expected behavior is that the block from ClassA::TestA() is correctly copied, and no warning generated.

**Version**
This works in 1.9.4 but not in 1.9.5 and later. Was introduced by commit b290399

**Comment by albert-github** [comment]
To be able to reproduce it the name of the file is missing as well as the doxygen settings file (Doxyfile), so:
- Can you please attach a, small, self contained example (source+configuration file in a, compressed, tar or zip file!) that allows us to reproduce the problem? Please don't add external links as they might not be persistent (also references to GitHub repositories are considered non persistent).

**Comment by EilifTS** [comment]
Here is a zip with a Doxyfile and reproducer: 
[reproducer.zip](https://github.com/user-attachments/files/16469691/reproducer.zip)

**Comment by albert-github** [comment]
It looks like that:
```
   namespace ClassA = NamespaceA::NamespaceB;
```
in
```
namespace NamespaceC
{
    namespace ClassA = NamespaceA::NamespaceB;
}
```
causes the problem, looks like the namespace alias (https://en.cppreference.com/w/cpp/language/namespace_alias) is not handled properly when the names of the alias is also the name of a (non related) class.

**Comment by doxygen** [comment]
@EilifTS Please verify if the referenced commit fixes the problem for you. Do not close the issue, this will be done automatically when the next official release becomes available.

**Comment by EilifTS** [comment]
Yes, the patch does indeed fix the issue. Thank you!

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.13.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
